src/db/rdb/models/user.model.ts

```diff
@@ -57,7 +58,12 @@ UserModel.init({
     token: {
         type: DataTypes.STRING,
         allowNull: true
-    }
+    },
+    images: {
+      type: DataTypes.JSON,
+      allowNull: true,
+      defaultValue: null,
```

Comment:
defaultValue doesn't need to be null. if it's not initialized and optional by default it will be null.